fern/apis/version-2024-11-13/definition/stt.yml

```
@@ -1,5 +1,219 @@
 # yaml-language-server: $schema=https://raw.githubusercontent.com/fern-api/fern/main/fern.schema.json
 
+service:
+  base-path: /stt
+  auth: true
+  endpoints:
+    transcribe:
+      path: ""
+      method: POST
+      display-name: Speech to Text (Batch)
```

we should use this throughout docs instead of "Speech to Text"

```suggestion
      display-name: Speech-to-Text (Batch)
```